notebooks/0.data_splits/data_splits.ipynb

```diff
@@ -0,0 +1,2808 @@
+{
```

Comment:
<p>I think plate holdout should be the first holdout. As I understand it now a plate could be held out that has some wells or even whole treatments removed from it. </p><p> </p><p>Consider removing holdout from the whole dataset in the following order:</p><ul><li>Plate</li><li>Treatment</li><li>Well</li></ul><p> </p><p>I think this might be a good approach but please let me know if I am missing something or if you have other thoughts!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/Cytotoxic-Nuisance-Metadata-Analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>